### DISCUSSION TITLE
AIrflow + Sentry Performance Monitoring

### DISCUSSION THREAD

**Discussion by lewismc** [apache/airflow#27968] [Q&A] [answered]
Hi Folks,
We have been using the 'basic' Sentry integration to better catalog and manage operational anomalies for our Airflow deployment in K8s.
We are NOT yet using [performance monitoring](https://docs.sentry.io/platforms/python/guides/flask/performance/) and wondered whether other community members are?
The same goes for [profiling](https://docs.sentry.io/platforms/python/guides/flask/profiling/).
I suppose that these are some more advanced aspects of the Sentry platform so we would like to understand if and how others are using them before we dive in the deep end.
Thanks for any info

**Comment by potiuk** [comment, marked as answer]
Just as a comment - in the future when we implement AIP-49: https://cwiki.apache.org/confluence/display/AIRFLOW/AIP-49+OpenTelemetry+Support+for+Apache+Airflow you might get more support from generic way how Open Telemetry provides the features of performance monitoing and profiling.

But no-one yet attempted to implement the AIP - so it will take quite some time to get it (unless you and other people would like to take on the task and start implementing it).

**Comment by lewismc** [comment]
Thank you @potiuk
